### PR TITLE
Quadrat: Add Code block styles

### PR DESCRIPTION
**Pull request by danieldudzic**
Until this WordPress/gutenberg#31135 gets resolved, I have added a temporary fix in Blank Canvas Blocks: #3704 which is required for this PR to work

#3640

### CODE REVIEW

File: quadrat/child-experimental-theme.json [hunk 1]
```diff
@@ -126,6 +127,11 @@
 						"left": "calc( 2 * var(--wp--custom--margin--horizontal) )"
 					}
 				},
+				"code": {
+					"typography": {
+						"fontFamily": "unset"
```
Comment (scruffian) [inline quadrat/child-experimental-theme.json]: Do we need this?

Comment (scruffian) [reply quadrat/child-experimental-theme.json]: Yes

**Comment by scruffian** [comment]
I removed some unneeded declarations from the child-theme.json.

**Comment by pbking** [comment]
I'm not sure I follow the font family part of this change...

The linked gutenberg issue notes that the code block's font family cannot be changed with styles, and while that does seem to be true the Figma Design doesn't seem to express something other than the browser's default monospace (rather than the base font which seems to be applied).

From what I can tell neither the Gutenberg fix or the BCB fix should be necessary to achieve the[ intended design.](https://www.figma.com/file/EDoXTvsZt4lMHItzOczFQw/Quadrat?node-id=50%3A766)

image

**Comment by danieldudzic** [comment]
> I'm not sure I follow the font family part of this change...
> 
> The linked gutenberg issue notes that the code block's font family cannot be changed with styles, and while that does seem to be true the Figma Design doesn't seem to express something other than the browser's default monospace (rather than the base font which seems to be applied).
> 
> From what I can tell neither the Gutenberg fix or the BCB fix should be necessary to achieve the[ intended design.](https://www.figma.com/file/EDoXTvsZt4lMHItzOczFQw/Quadrat?node-id=50%3A766)
> 
> image

What browser are you using? I noticed the issue in Chrome.

**Comment by pbking** [comment]
I'm using chrome for development.  

On this branch I see the below:
image

Which is not correct for the intended design (noted above).

If I remove all of the typeface changes from quadrat's theme.json then it looks more correct:
image

However, the fix to adjust the typeface isn't needed in that situation, since the default is correct (I believe).

**Comment by pbking** [comment]
I cleaned out this branch and cherry-picked only the important things.
I also removed the font-related theme.json settings to get the monospace font back per the design.
I think this is ready for another look-see.

**Review by scruffian — APPROVED** [review]
LGTM

**Pull request merged**